src/main/java/edu/phema/elm_to_omop/vocabulary/translate/map/ActCodeToVisitConceptMap.java

```diff
@@ -0,0 +1,45 @@
+package edu.phema.elm_to_omop.vocabulary.translate.map;
+
+import edu.phema.elm_to_omop.vocabulary.phema.PhemaCode;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Translates the ActCode vocabulary to the OMOP internal visit concept vocabulary
+ * <p>
+ * Source: https://www.hl7.org/fhir/STU3/v3/ActEncounterCode/vs.html
+ * Target: http://athena.ohdsi.org/search-terms/terms?vocabulary=Visit
+ */
+public class ActCodeToVisitConceptMap implements IVocabularyMap {
+    private static Map<String, String> map;
+
+    static {
+        map = new HashMap<>();
+
+        // TODO: Have someone validate these mappings
```

Comment:
I have a few notes in there about the mappings.  I think these are reasonable starts, but we should discuss with the broader group and maybe even OHDSI community to see what mappings make sense.

Reply:
Agreed it needs proper review, but it probably fine for our current use case.